app/src/main/res/values/help.xml

```diff
@@ -5,56 +5,49 @@
+stars, constellations, and planets in that part of the sky.&lt;br/&gt;Next time  what to identify a bright star or planet, use Sky Map!&lt;br/&gt;
 
 &lt;b&gt;Visible Layers&lt;/b&gt;&lt;br/&gt;
-
 Sky Map organizes the objects it shows you into layers.  Each layer can
 be toggled using the on-screen controls.  The layers are:&lt;br/&gt;
-      Stars&lt;br/&gt;
-      Constellations&lt;br/&gt;
-      Messier Objects&lt;br/&gt;
-      Planets&lt;br/&gt;
-      Meteor Showers&lt;br/&gt;
-      Right Ascension and Declination Grid&lt;br/&gt;
-      Horizon and cardinal points&lt;br/&gt;&lt;br/&gt;
-The brightest stars and most prominent constellations are labeled, as are the
-planets.  The planets are displayed as images that are larger than
-actual size.  The horizon is represented as a straight line.  Unless you live
-somewhere that is completely flat and free of buildings, it is likely that
-Sky Map will show you objects above the horizon line that you are actually
-unable to see.&lt;br/&gt;&lt;br/&gt;
+      <span>&#8226;</span>Stars&lt;br/&gt;
+      <span>&#8226;</span>Constellations&lt;br/&gt;
+      <span>&#8226;</span>Messier Objects&lt;br/&gt;
+      <span>&#8226;</span>Planets&lt;br/&gt;
+      <span>&#8226;</span>Meteor Showers&lt;br/&gt;
+      <span>&#8226;</span>Right Ascension and Declination Grid&lt;br/&gt;
+      <span>&#8226;</span> Horizon and cardinal points&lt;br/&gt;&lt;br/&gt;
+      
+The brightest stars, constellations, and planets are labeled. 
+Planets display as images that are larger than will appear in the sky.
+Sky Map may show objects that are above the horizon line (represented as a straight line)
+that you may not be able to use, unless you live in an area that is  flat and without buildings.&lt;br/&gt;&lt;br/&gt;
 
 &lt;b&gt;On-screen controls&lt;/b&gt;&lt;br/&gt;
-Tap on the screen to show the on-screen controls.  On one side of the screen are
+Tap on the screen to show the on-screen controls. On the side of the screen are
 the toggles for the layers described above.  They are orange when the layers are
 enabled and white when disabled.&lt;br/&gt;
 
-Along the bottom of the screen are zoom controls, and a toggle to switch between
+The bottom of the screen displays zoom controls and a toggle to switch between
 automatic and manual mode.&lt;br/&gt;&lt;br/&gt;
 &lt;b&gt;Navigation Modes&lt;/b&gt;&lt;br/&gt;
-When first installed Sky Map is in Automatic Mode and uses the phone\'s sensors
-to show you a map of the sky in the direction you are holding the phone.  By
+Sky Map installs in Automatic Mode and uses the phone\'s sensors
+to display a map of the sky in the direction your phonw is pointing. By
 switching into Manual Mode, you can drag the map with your finger to explore the
-sky.  Move the trackball or D-pad left and right to rotate the map.  On multitouch phones
+sky. To rotate the map left or right, move the trackball or D-pad. On multitouch phones
```

Comment:
Hah - this made me laugh. It shows how long since we updated the documentation.  Who has a phone with a D-pad these days? Can you delete that part of it and just explain how to rotate the map with your fingers?